pkg/api/registered/registered.go

```diff
@@ -72,6 +74,11 @@ func init() {
 	RegisteredGroupVersions = supportedVersions
 }
 
+// ValidateGroupVersion declares a groupVersion as valid.
+func ValidateGroupVersion(groupVersion unversioned.GroupVersion) {
```

Comment:
also, `init()` is the thing that uses `validGroupVersions`... won't adding another valid group version always happen too late?